pinot-tools/src/main/java/org/apache/pinot/tools/Quickstart.java

```diff
@@ -91,30 +114,17 @@ public static String prettyPrintResponse(JsonNode response) {
 
   public void execute()
       throws Exception {
+    String tableName = getTableName();
```

Comment:
Suggest keep the logic here for quickstart and load extra data dir in QuickStartBase.

Reply:
Refactored a bit to pass default directory path as argument to function in QuickStartBase.